scripts/convertConfig.ts

```diff
@@ -62,7 +62,7 @@ export type ConfigData = {
     name?: string;
     mainFeatures?: any;
     fonts?: {
-        name: string;
+        name?: string;
```

Comment:
@7dev7urandom Should I set this as optional to document that it might not be there?